interop-testing/src/main/java/io/grpc/testing/integration/StressTestClient.java

```diff
@@ -308,7 +308,9 @@ void shutdown() {
     }
 
     try {
-      metricsServer.shutdownNow();
+      if (metricsServer != null) {
+        metricsServer.shutdownNow();
+      }
```

Comment:
Since this is unrelated to the change, could we please move it to a separate PR? To avoid any confusion, and in case we need to roll back this PR.